### PR TITLE
CL-1311: Added examples in Javascript and PHP

### PR DESCRIPTION
**Pull request by pola**

### CODE REVIEW

File: javascript/index.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+'use strict'
+
+const fs = require('fs')
```
Comment (davidjungermann) [inline javascript/index.js:3]: Any particular reason for not using ES6 modules here?

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,4 @@
 \#*
 .DS_Store
```
Comment (davidjungermann) [inline .gitignore]: As Fredrik made me aware of yesterday, this maybe should be removed? It is not related to the code itself, and should rather be added to a global .gitignore file for MacOS users.

File: javascript/index.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+const express = require('express')
+const app = express()
+const bodyParser = require('body-parser')
+
+const port = 1234
+
+const timestamp = () => Math.floor(+new Date() / 1000)
+const randomNumber = (min, max) => Math.floor(Math.random() * (max - min + 1) + min)
+
+app.use(bodyParser.json())
```
Comment (davidjungermann) [inline javascript/index.js]: I get a warning that the bodyParser module is deprecated.

File: javascript/index.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+	const postData = req.body
+	const deviceId = postData.deviceId
+	
+	if (/^(D[A-F0-9]{16})|(G[A-F0-9]{18})$/.test(deviceId)) {
+		console.log('Saving data from device ' + deviceId)
+		
+		const folder = 'data/' + deviceId
+		const file = folder + '/' + timestamp() + '_' + randomNumber(10000, 99999) + '.json'
+		
+		await fs.mkdirSync(folder, { recursive: true })
```
Comment (davidjungermann) [inline javascript/index.js]: I'm getting this tooltip: `await` has no effect on the type of this expression.ts(80007)

**Review by davidjungermann — COMMENTED** [review]
Good examples! I added some minor comments.

Maybe it also would be nice with a quick README for each example, that briefly goes over what the example does, and how to run it.

File: javascript/index.js (hunk 1 above)
Comment (pola) [reply javascript/index.js:3]: The ES6 notation for importing packages is not as compatible as this one.
https://stackoverflow.com/questions/45854169/how-can-i-use-an-es6-import-in-node-js

**Review by davidjungermann — APPROVED** [review]
🌟

**Pull request merged**